<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom joints</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../icons.qrc">
    <normaloff>:/icons/collection-triangular-iteration.png</normaloff>:/icons/collection-triangular-iteration.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="new_group">
     <property name="title">
      <string>New joints</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QListWidget" name="custom_list"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="link_choose_label">
          <property name="text">
           <string>Belong with:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="link_choose"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="add_button">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_button">
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../icons.qrc">
            <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="multiple_group">
     <property name="title">
      <string>Multiple joints</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="multiple_list"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="quote_choose_label">
          <property name="text">
           <string>Quoted from:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="quote_choose"/>
        </item>
        <item>
         <widget class="QLabel" name="quote_link_choose_label">
          <property name="text">
           <string>Belong with:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="quote_link_choose"/>
        </item>
        <item>
         <widget class="QLabel" name="joint_choose_label">
          <property name="text">
           <string>Joint:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="joint_choose"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="add_mj_button">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_mj_button">
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../icons.qrc">
            <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
